.pre-commit-config.yaml

```diff
@@ -76,22 +77,22 @@ repos:
     -   id: nbstripout
         files: ".ipynb"
 
--   repo: https://github.com/mwouts/jupytext
-    rev: v1.11.3
-    hooks:
-    -   id: jupytext
-        files: '^examples/tutorials/(colabs|nb_python)/(.*\.py|.*\.ipynb)$'
-        args: [--update-metadata, '{"jupytext":{"notebook_metadata_filter":"all", "cell_metadata_filter":"-all"}, "accelerator":"GPU"}', --set-formats, 'nb_python//py:percent,colabs//ipynb,', --pipe, black, --pipe, 'isort - --treat-comment-as-code "# %%"', --pipe-fmt, 'py:percent', --sync]
-        additional_dependencies:
-            - 'nbformat<=5.0.8'
-            - black
-            - isort
+#-   repo: https://github.com/mwouts/jupytext
+#    rev: v1.11.3
+#    hooks:
+#    -   id: jupytext
+#        files: '^examples/tutorials/(colabs|nb_python)/(.*\.py|.*\.ipynb)$'
+#        args: [--update-metadata, '{"jupytext":{"notebook_metadata_filter":"all", "cell_metadata_filter":"-all"}, "accelerator":"GPU"}', --set-formats, 'nb_python//py:percent,colabs//ipynb,', --pipe, black, --pipe, 'isort - --treat-comment-as-code "# %%"', --pipe-fmt, 'py:percent', --sync]
+#        additional_dependencies:
+#            - 'nbformat<=5.0.8'
+#            - black
+#            - isort
 
--   repo: https://github.com/shellcheck-py/shellcheck-py
-    rev: v0.7.2.1
-    hooks:
-    -   id: shellcheck
-        exclude: ^habitat_baselines/slambased/
+#-   repo: https://github.com/shellcheck-py/shellcheck-py
```

Comment:
Shellcheck might help use prevent a few bugs though.

Reply:
You're right, I'll add this back.